ci/checkLinks.ts

```diff
@@ -0,0 +1,185 @@
+import { readdirSync, statSync, readFileSync } from "fs";
+import path from "path";
+import chalk from "chalk";
+import * as github from "@actions/core";
+const cwd = process.env.GITHUB_ACTIONS ? process.env.GITHUB_WORKSPACE! : process.cwd();
+
+function importDirectory(directory: string, extension: string, subdirectories = true) {
```

Comment:
```suggestion
function importDirectory(directory: string, extension: string, subdirectories: boolean = true) {
```
Wouldn't it be more consistent if it were like the other parameters?

Reply:
Its unnecessary and against the eslint rules (yes, I know I added the rules)
![image](https://user-images.githubusercontent.com/8060149/136337456-9701e4cb-1faf-486e-be57-2d02bf7d0724.png)

Reply:
My bad, you actually used a preset so I didn't see the rule. Moving forward...

Reply:
~~Whilst that seems like it would be a good idea, the ESLint config used disallows this.
![unknown](https://user-images.githubusercontent.com/17235016/136339142-1a836bac-3629-4adf-83ad-fcea65128fa4.png)~~
Ignore, GitHub didn't update comments